actors/util/adt/map.go

```diff
@@ -153,7 +154,7 @@ func (r rtStore) Context() context.Context {
 
 func (r rtStore) Get(ctx context.Context, c cid.Cid, out interface{}) error {
 	if !r.IpldGet(c, out.(vmr.CBORUnmarshaler)) {
-		r.AbortStateMsg("not found")
+		r.Abort(exitcode.ErrNotFound, "not found")
```

Comment:
This should be illegal state, it means something is messed up in internal state.